opslevel/import_utils.go

```diff
@@ -0,0 +1,59 @@
+package opslevel
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/opslevel/opslevel-go/v2024"
+)
+
+func extractContactFromContacts(contactId opslevel.ID, contacts []opslevel.Contact) *opslevel.Contact {
+	for _, readContact := range contacts {
+		if contactId == readContact.Id {
+			return &readContact
+		}
+	}
+	return nil
+}
+
+func extractTagFromTags(tagId opslevel.ID, tags []opslevel.Tag) *opslevel.Tag {
+	for _, readTag := range tags {
+		if tagId == readTag.Id {
+			return &readTag
+		}
+	}
+	return nil
+}
+
+func extractToolFromTools(toolId opslevel.ID, tools []opslevel.Tool) *opslevel.Tool {
+	for _, readTool := range tools {
+		if toolId == readTool.Id {
+			return &readTool
+		}
+	}
+	return nil
+}
+
+func getTagsFromResource(client *opslevel.Client, resource opslevel.TaggableResourceInterface) (*opslevel.TagConnection, diag.Diagnostics) {
+	var diags diag.Diagnostics
+	tags, err := resource.GetTags(client, nil)
+	if err != nil {
+		diags.AddError(
+			"opslevel client error",
+			fmt.Sprintf("unable to read tags on %s with id '%s', got error: %s", string(resource.ResourceType()), string(resource.ResourceId()), err),
+		)
+	}
+	if tags == nil {
+		diags.AddError(
```

Comment:
Why does this result in a diagnostic error?

Reply:
To match the function signature :)